test-resources/msgpack_test_fixtures.json

```diff
@@ -0,0 +1,75 @@
+[
```

Comment:
I think we should document in the feature spec (specifically in [RSL6a1](https://sdk.ably.com/builds/ably/specification/main/features/#RSL6a1)) how we expect client libraries to use these fixtures (and perhaps put a comment in this file linking to it).

Reply:
OK. But that should be in a separate PR.

Reply:
Yes the feature spec is a separate repo so would need a separate PR, but I think it's worth doing as part of the work your doing so other SDK developers know these fixtures exist and how to test against them.